node/resolvers/Routes/utils/index.ts

```diff
@@ -123,6 +123,7 @@ export const generateClUser = async ({
 
   if (businessName && businessDocument) {
     clUser.isCorporate = true
+    clUser.document = null
```

Comment:
I don't understand this change, what's the difference between `document` and `corporateDocument`, and why do we need to set this one to null?